InWorldz/InWorldz.VivoxVoice/VivoxVoiceModule.cs

```diff
@@ -759,7 +759,7 @@ private string RegionGetOrCreateChannel(Scene scene, LandData land)
             else
             {
                 landName = String.Format("{0}:{1}", scene.RegionInfo.RegionName, scene.RegionInfo.RegionName);
-                landUUID = m_forcedChannelName == String.Empty ? scene.RegionInfo.RegionID.ToString() : m_forcedChannelName;
+                landUUID = String.IsNullOrEmpty(m_forcedChannelName) ? scene.RegionInfo.RegionID.ToString() : m_forcedChannelName;
```

Comment:
No semantic change: The only assignment to `m_forcedChannelName` is guaranteed to not be null: line 147 `m_forcedChannelName = m_config.GetString("forced_channel_name", String.Empty).ToLower();`